tests/plugins/test_download_remote_source.py

```diff
@@ -125,12 +130,23 @@ def test_download_remote_source(
                 plugin.run()
             return
 
+        # Remote source folder creation for cachito.env file
+        abs_path_remote_source_dir = os.path.join(plugin.workflow.builder.df_dir,
+                                                  plugin.REMOTE_SOURCE)
+        if os.path.isdir(abs_path_remote_source_dir):
+            os.makedirs(abs_path_remote_source_dir)
```

Comment:
I don't understand this one, are we checking if the dir exists and if yes , we are creating it again?

Reply:
Yeah that's wrong test passed only because folder was already created. OK removing this part. Thanks!